src/load.ts

```diff
@@ -78,6 +78,23 @@ export async function load(
     }
 }
 
+/**
+ * Loads the data from a CDN and returns an instance of AzureAppConfiguration.
+ * @param endpoint  The URL to the CDN.
+ * @param options  Optional parameters.
+ */
+export async function loadFromCdn(endpoint: URL | string, options?: AzureAppConfigurationOptions): Promise<AzureAppConfiguration>;
+
+export async function loadFromCdn(
+    endpoint: string | URL,
+    appConfigOptions?: AzureAppConfigurationOptions
+): Promise<AzureAppConfiguration> {
+    const emptyTokenCredential: TokenCredential = {
+        getToken: async () => ({ token: "", expiresOnTimestamp: 0 })

```

Comment:
do we need to override the sync version?

Reply:
![image](https://github.com/user-attachments/assets/71912195-ea34-425d-bbf6-52c51d27924b)

The `TokenCredential` interface requires getToken to be async